include/gc/Transforms/Passes.td

```diff
@@ -32,7 +32,7 @@ def ConvertOneDNNGraphToLinalg : Pass<"convert-onednn-graph-to-linalg"> {
   ];
 }
 
-
+#ifdef GC_USE_GPU
```

Comment:
@dchigarev Please help to view this change. Thx!. This is to totally disable xegpu passes when GPU is OFF. Just to make compiler & CI happy for CPU mode.

Reply:
I'm good with this change